File: src/Web/Spa/src/app/models/school.ts
export class School{
    public ID:string;
    public Name:string;
    public StreetName:string;
    public HouseNumber:number;
    public PostalCode:string;
    public City:string;
    public Country:string;

    public constructor(id:string,name:string,street:string,houseNumber:number,postalCode:string,city:string,country:string){
        this.ID=id;
        this.Name=name;
        this.StreetName=street;
        this.HouseNumber=houseNumber;
        this.PostalCode=postalCode;
        this.City=city;
        this.Country=country;
    }
}

File: src/Services/Identity/Identity.EventHandler/Models/School.cs
using System;
using System.Collections.Generic;

namespace Identity.EventHandler.Models
{
    public partial class School
    {
        public School()
        {
            Programs = new HashSet<Program>();
        }

        public string Id { get; set; }
        public string Name { get; set; }
        public string StreetName { get; set; }
        public int? HouseNumber { get; set; }
        public string PostalCode { get; set; }
        public string City { get; set; }
        public string Country { get; set; }

        public virtual ICollection<Program> Programs { get; set; }
    }
}
